include/SugarFields/Fields/Multienum/SugarFieldMultienum.php

```diff
@@ -58,7 +58,7 @@ function getSearchViewSmarty($parentFieldArray, $vardef, $displayParams, $tabind
     	}
     }
 
-    function displayFromFunc( $displayType, $parentFieldArray, $vardef, $displayParams, $tabindex ) {
+    function displayFromFunc( $displayType, $parentFieldArray, $vardef, $displayParams, $tabindex = 0) {
```

Comment:
So this because you are change signature here, it would be a good idea to also change the signature of the parent variations of this method.

Can you please add PHP Doc above this method with the "type" @params ... as it helps developers IDE's detect if a problem occurred.

Reply:
@daniel-samson I confess my approach to this was rather superficial - I see a lot these problems in the forums, and I learned to fix them simply matching the two declarations in the error message.

Reply:
I don't know the code well enough to document parameters in PHP Doc - anyway I touched one declaration, but all of them are lacking this.

Reply:
(crap, shift-enter here behaves like enter! sorry for the multiple messages)  I can't find any other method with this signature that needs fixing, I searched the code for `displayFromFunc` and it only appears here, apart from some places where the function is called.

Reply:
I think the parents already have the default value for the $tabindex parameter. That was the reason the warning was appearing in the first place.

My IDE suggests the barebone phpdoc for this function should be:
`
    /**
     * @param $displayType
     * @param $parentFieldArray
     * @param $vardef
     * @param $displayParams
     * @param int $tabindex
     * @return string
     */`

Reply:
I just added that PHP Doc. For what it's worth... which isn't much. This is not a good time for me to invest in tidying up the file, sorry. If somebody else wants to and has some minutes available, please go ahead. Other than that, I think this is a good PR to try and put into the next release anyway: it's one less bug.